### PR TITLE
Replace issue templates with new issue forms

### PR DESCRIPTION
**Pull request by NivekT**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #115

Differential Revision: [D32883988](https://our.internmc.facebook.com/intern/diff/D32883988)

### CODE REVIEW

**Comment by NivekT** [comment]
@nivekt has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32883988).

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
+      # For security purposes, please check the contents of collect_env.py before running it.
+      python collect_env.py
+      ```
+  validations:
+    required: true
+- type: markdown
+  attributes:
+    value: >
+      Thanks for contributing 🎉!
```
Comment (ejguan) [inline .github/ISSUE_TEMPLATE/bug-report.yml:57]: Can you please fix `No newline at end of file`?

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+- type: textarea
+  attributes:
+    label: Versions
+    description: |
+      Please run the following and paste the output below.
+      ```sh
+      wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
+      # For security purposes, please check the contents of collect_env.py before running it.
+      python collect_env.py
+      ```
```
Comment (ejguan) [inline .github/ISSUE_TEMPLATE/bug-report.yml:51]: We should also include the version of torchdata in the future.

File: .github/ISSUE_TEMPLATE/documentation.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+name: 📚 Documentation
+description: Report an issue related to https://pytorch.org/docs/stable/index.html
+
+body:
+- type: textarea
+  attributes:
+    label: 📚 The doc issue
+    description: >
+      A clear and concise description of what content in https://pytorch.org/docs/stable/index.html is an issue. If this has to do with the general https://pytorch.org website, please file an issue at https://github.com/pytorch/pytorch.github.io/issues/new/choose instead. If this has to do with https://pytorch.org/tutorials, please file an issue at https://github.com/pytorch/tutorials/issues/new.
```
Comment (ejguan) [inline .github/ISSUE_TEMPLATE/documentation.yml:9]: Emmm. The description here is incorrect for TorchData. And, we only have inline doc for now.

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 2 above)
Comment (NivekT) [reply .github/ISSUE_TEMPLATE/bug-report.yml:51]: I was under the impression that if they run the `collect_env.py` it should list out the package version too but I can certainly add that to make it more explicit.

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 1 above)
Comment (NivekT) [reply .github/ISSUE_TEMPLATE/bug-report.yml:57]: Done

File: .github/ISSUE_TEMPLATE/documentation.yml (hunk 3 above)
Comment (NivekT) [reply .github/ISSUE_TEMPLATE/documentation.yml:9]: I will update it and let users know that we only have inline docs for now and the web version is coming soon.

**Comment by NivekT** [comment]
@nivekt has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32883988).

**Review by ejguan — APPROVED** [review]
LGTM, really appreciate the work!

**Pull request closed without merging**